rate_counter/rate/tasks.py

```diff
@@ -96,3 +104,37 @@ def get_url(inst='1', group='', semestr='1', semestr_part='1'):
                                                                                                 semestr,
                                                                                                 semestr_part,
                                                                                                 )
+
+
+def rate_graphs(id):
+    '''
+    prepare data for json column charts
+    '''
+    get_data = TableRow.objects.filter(table_id=id)
+    list = []
+    tmp=[]
+    tmp_name=[]
+    for line in get_data:
+        tmp_name.append(line.name)
+        tmp.append(line.point)
+    list.append(tmp_name)
+    list.append(tmp)
+    x = 'Rate Graphs'
+    y = '100'
+    list[0][0]=x
```

Comment:
```
list[0][0]=x
list[1][0]=y
```
should be changed to 
```
list[0][0] = x
list[1][0] = x
```
PEP8 issue - there should be space around operator. '=' in this case